PlusContent/privacy-mode.md

```diff
@@ -1 +1,31 @@
-The privacy mode prevents sandboxed processes from reading non-generic Windows paths, in this way protecting confidential user data from being accessed by sandboxed processes.
\ No newline at end of file
+# Privacy Mode
+
+**NOTE: This feature requires a [supporter certificate.](https://sandboxie-plus.com/supporter-certificate/)**
+
+The concept of privacy mode and privacy enhanced (or Data Protection) boxes was introduced in **Sandboxie Plus v1.0.0**.
+In this mode, most of the locations on a PC are set to be treated like a Write[File/Key]Path
+meaning the sandbox locations are writable but the unsandboxed locations are not readable.
+The disk "user space" appears empty! In addition, the registry does not allow reading of user root keys. In other words, even though sandboxed processes can continue to work, they cannot access private user data.
+
+The setting for a privacy enhanced box can be enabled by adding `UsePrivacyMode=y` to the box settings section of **Sandboxie.ini**. It can also be enabled in the Sandman UI. Right-click on a box and select "Sandbox Options" from the drop-down menu (or simply double-click on a box) to bring up the Box Options UI. Select the Box Type Preset as "Sandbox with Data Protection" (with a **blue** box icon) and click OK to apply changes. It is sometimes called a **blue box** in various security forums. The status column of Sandman UI labels this box as **Privacy Enhanced**.
+
+![](../Media/Box_PrivacyMode.png)
+
+
+**What is User Space?** AppGuard refers to [user space](https://malwaretips.com/threads/run-by-smartscreen-utility.65145/post-561364) as "computer storage space that is typically accessible by non-admin Windows users. It contains the user's profile directory (which includes the My Documents folder and Desktop), removable storage devices, network shares, and all non-system hard drives such as additional external and internal disk drives." Think of "user space" as everything outside the **system** (where the core operating system and programs live), in other words, outside the `C:\Windows`, `C:\Program Files`, and `C:\Program Files (x86)` folders!
+
+Internally, a privacy enhanced box is based on three defaults:
+1. **Allow read access to system resources:**
+- `C:\Windows`
+- `C:\Program Files`
+- `C:\Program Files (x86)`
+- HKLM (but not HKCU) registry keys using `ReadKeyPath=HKEY_LOCAL_MACHINE*`
+- **Note:** The read access provides a good balance between privacy and convenience. One could, of course, drill down to identify selected system resources that may leak private data and further restrict them (using `Write[File/Key]Path`) if desired.
+
+2. **Hide (and block access to) user space:**
+- In user space, a privacy box works in **default block** mode: every path is a Write[File/Key]Path (which does not allow read/write changes to sandbox) unless specifically allowed by an over-riding rule. Access to selected paths is enabled by invoking [Rule Specificity](../PlusContent/RuleSpecificity.md).
```

Comment:
> which does not allow read/write changes to sandbox

which hides all files and folders outside the sandbox, but allows new files and folders to be created in the sandbox.

Reply:
Will fix.